pkgs/tools/security/cliam/default.nix

```diff
@@ -0,0 +1,34 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+}:
+
+buildGoModule rec {
+  pname = "cliam";
+  version = "1.0.0";
+
+  src = fetchFromGitHub {
+    owner = "securisec";
+    repo = pname;
+    rev = version;
+    hash = "sha256-bq7u6pknokyY4WwO1qMYPuY86UZlDgeYEa1AJpk8d+4=";
+  };
+
+  vendorSha256 = "sha256-aGBA97EvIUv9myqcrtltiVxh1/0VtrQy2j9GU6r197g=";
+
+  ldflags = [
+    "-s -w"
+    "-X github.com/securisec/cliam/cli/version.Version=${version}"
+  ];
+
+  postInstall = ''
+    mv $out/bin/cli $out/bin/$pname
+  '';
+
+  meta = with lib; {
+    description = "Cloud agnostic IAM permissions enumerator";
+    homepage = "https://github.com/securisec/cliam";
+    license = licenses.gpl3Only;
```

Comment:
do we need to check if this is Only or Plus?

Reply:
There is no mentioning of "any later...". Thus, we assume that it's "Only".